types/crew.model.ts

```diff
@@ -0,0 +1,64 @@
+import { Club } from './club.model';
+import { DisplayStatus } from './display-status.model';
+import { Event } from './event.model';
+import { Result } from './result.model';
+import { Rower } from './rower.model';
+
+export interface Crew {
+	id: string;
+	club_id: string;
+	regatta_id: string;
+	event_id: string;
+	sub_event_id: null; // Wat is een sub_event?
+	subEventCode: string;
+	knrb_num: number;
+	knrb_sequence: number; // Wat is de sequence?
+	status: number;
+	result_status: null;
+	clubName: string;
+	displayName: string; // Wat is het verschil tussen 'displayName' en 'name'?
```

Comment:
displayName is de naam die weer geven word. Dit is afhankelijk van wedstrijd instellingen maar ongeveer:

if (alternative) displayName = alternative
elseif (sequence) displayName = club_name + sequence
else displayName = club_name

Wat in de name staat is volgens mij de clubname weet ik niet zeker.

Reply:
Graag alleen de displayName gebruiken

Reply:
Is de 'clubName' dan nog nodig als het hetzelfde is als 'name'?

Reply:
clubname word op de PDF getoond voor de head of the river bijvoorbeeld. Bij de PEiL is het de display name

Reply:
Ja oké, maar er is nu 'name', 'clubName' en 'displayName'

Reply:
name is niet relevant.

clubName is voor de wedstrijden als head of the river
displayName is voor studenten compementie wedstrijden